src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +14,16 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService userService = new UserService();
+        User user = new User(email, password);
+        if (userService.findByEmail(email) == null) {
+            return false;
+        }
+        if (user.getEmail().equals(userService.findByEmail(email).getEmail())
+                && user.getPassword() == userService.findByEmail(email).getPassword()) {
+            return true;
+        } 
         return false;
```

Comment:
- we should remove first part of condition
- we should use equals() instead of ==
```suggestion
        return password.equals(user.getPassword());
```